docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -307,9 +307,16 @@ You can make further modifications to a saved filter view later using kebab menu
 
 ## Create an SLO from Metrics page
 
-:::note Coming soon
-You'll be able to create SLOs from Metrics Explorer.
-:::
+To create an SLO from the **Metrics** page:
+
+1. Go to **Manage Data** > **Metrics**.
```

Comment:
```suggestion
1. Open a new or go to an existing **Metrics** tab.
```

Reply:
@apoorv-garg is this what you meant? 

_Click **+ New** > **Metrics** or go to an existing **Metrics** tab._

Reply:
Yes